libs/site-base/src/containers/AddAccount/CreateExistingSwitch/index.tsx

```diff
@@ -47,13 +42,23 @@ export const CreateExistingSwitch = (props: Props) => {
         <Header as="h1" className={` x12spaceAfter`}>
           <FormattedMessage {...translations.title} />
         </Header>
-        <ButtonPrimary as={NavLink} to={createUrl} className={styles.button}>
+        <ButtonPrimary as={NavLink} to={`${url}generate/intro`} className={styles.button}>
           <FormattedMessage {...translations.buttonNewAccount} />
         </ButtonPrimary>
         <Divider horizontal>Or</Divider>
-        <ButtonSecondary as={NavLink} to={existingUrl} className={styles.button}>
+        <ButtonSecondary as={NavLink} to={`${url}restore/`} className={styles.button}>
           <FormattedMessage {...translations.buttonAccount} />
         </ButtonSecondary>
+        <div className={`x8spaceBefore`}>
+            <Link className={styles.linkEthereum} to={`${url}connect/create`}>
+              <FormattedMessage {...translations.ethereumNewAccount} />
+            </Link>
+        </div>
+        <div>
+            <Link className={styles.linkEthereum} to={`${url}connect/exist`}>
+              <FormattedMessage {...translations.ethereumExistingAccount} />
+            </Link>
+        </div>
```

Comment:
Wasn't sure if both links are needed. Tell me if you want me to remove one of them.

Reply:
We'll leave both in for now.  We can wait until we see user behaviour to see if we want to reduce it at all